packages/augur-ui/src/modules/portfolio/components/portfolio-view/my-bets.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+
+import Styles from 'modules/portfolio/components/portfolio-view/my-bets.styles.less';
+import { ExternalLinkButton, PrimaryButton } from 'modules/common/buttons';
+import { PillSelection, SquareDropdown } from 'modules/common/selection';
+import FilterSearch from 'modules/filter-sort/containers/filter-search';
+import { HelmetTag } from 'modules/seo/helmet-tag';
+import { PORTFOLIO_VIEW_HEAD_TAGS } from 'modules/seo/helmet-configs';
+import { SPORTS_MARKET_TYPES, MY_BETS_VIEW_BY, MY_BETS_MARKET_STATUS } from 'modules/common/constants';
+import { MARKETS } from 'modules/routes/constants/views';
+import { FilterNotice } from 'modules/common/filter-notice';
+
+export const MyBets = () => {
```

Comment:
remove the return and wrapper brackets since it's not doing anything but returning the jsx